datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/mutate/util/SiblingsUtils.java

```diff
@@ -0,0 +1,32 @@
+package com.linkedin.datahub.graphql.resolvers.mutate.util;
+
+import com.linkedin.common.Siblings;
+import com.linkedin.common.urn.Urn;
+import com.linkedin.metadata.entity.EntityService;
+
+import javax.annotation.Nonnull;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static com.linkedin.metadata.Constants.SIBLINGS_ASPECT_NAME;
+
+public class SiblingsUtils {
+
+  private SiblingsUtils() { }
+
+  public static List<Urn> getSiblingUrns(@Nonnull final Urn entityUrn, @Nonnull final EntityService entityService) {
```

Comment:
Let's have a unit test for this static helps function!

Reply:
for sure! this and the other one